chall02/dapalmer.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+
+import sys
+
+def main():
+    if len(sys.argv) == 2:
+        morseCode = {
+            'A': '.-',
+            'B': '-...',
+            'C': '-.-.',
+            'D': '-..',
+            'E': '.',
+            'F': '..-.',   
+            'G': '--.',
+            'H': '....',
+            'I': '..',   
+            'J': '.---',
+            'K': '-.-',
+            'L': '.-..',   
+            'M': '--',
+            'N': '-.',
+            'O': '---',   
+            'P': '.--.',
+            'Q': '--.-',
+            'R': '.-.',
+            'S': '...',
+            'T': '-',
+            'U': '..-',
+            'V': '...-',
+            'W': '.--',
+            'X': '-..-',
+            'Y': '-.--',
+            'Z': '--..',
+            ' ': ' '       
+        }
+        origString = sys.argv[1].upper()
+        morseString = ''
+
+        for char in origString:
+            if char >= 'A' and char <= 'Z' or char == ' ':
+                morseString += morseCode[char]
+        print morseString
+    
+    else:
+        print 'Usage:', sys.argv[0], '<a-zA-Z>'
```

Comment:
[Python2 is deprecated](https://pythonclock.org/), you should use python3 as of January the 1st of 2020.

Also the usage you print does not correspond to the one expected in the subject.

```suggestion
        print('usage:', sys.argv[0], '<a-zA-Z string>')
```